hemres/templates/hemres/subscriptions_email.html

```diff
@@ -2,7 +2,7 @@
 {% load hemres_email %}
 {% block title %}Jonge Democraten Nieuwsbrieven{% endblock %}
 {% block content %}
-<div style="display: block; margin-bottom: 8px;"><img src="{% emailimage_static 'hemres/images/JongeDemocraten.png' %}" width="316" height="100px"  border="0" /></div>
+<div style="display: block; margin-bottom: 8px;"><img src="{% emailimage_static 'hemres/images/JongeDemocraten.png' %}" width="240px" height="100px"  border="0" /></div>
```

Comment:
Not sure about the intention here (limit width, height, both, or match true image size), but you might make it more flexible for future logo changes with something like this `style="max-height: 100px; width: auto;`. Just a note, might not apply and also not important.